core/dbt/tests/fixtures/project.py

```diff
@@ -454,5 +454,13 @@ def project(
 
     yield project
 
-    project.drop_test_schema()
+    # deps, debug and clean commands will not have an installed adapter when running and will raise
+    # a KeyError here.  Just pass for now.
+    # See https://github.com/dbt-labs/dbt-core/issues/5041
+    # The debug command also results in an AttributeError since `QueryCommentedProfile` doesn't have
+    # a `load_dependencies` method.
+    try:
+        project.drop_test_schema()
+    except (KeyError, AttributeError):
```

Comment:
@gshank I remember you had some changes for drop_test_schema in one of your PR, is that merged in? Is this still needed if your changes are merged?

Reply:
I changed create_schema and drop_schema to allow multiple schemas, but I think this try/except is still needed for this test. It would be good to figure out a more complete way of handling this, but I think this is okay for now.